App/Redux/ThreadsRedux.ts

```diff
@@ -28,6 +28,24 @@ const actions = {
   }),
   refreshThreadsError: createAction('REFRESH_THREADS_ERROR', resolve => {
     return (error: Error) => resolve({ error })
+  }),
+  addExternalInvite: createAction('ADD_EXTERNAL_THREAD_INVITE', resolve => {
+    return (name: string, pubKey: string) => resolve({ name, pubKey })
+  }),
+  addExternalInviteSuccess: createAction('ADD_EXTERNAL_THREAD_INVITE_SUCCESS', resolve => {
+    return (pubKey: string, link: string) => resolve({ pubKey, link })
+  }),
+  addExternalInviteError: createAction('ADD_EXTERNAL_THREAD_INVITE_ERROR', resolve => {
+    return (error: Error) => resolve({ error })
+  }),
+  acceptExternalInvite: createAction('ACCEPT_EXTERNAL_THREAD_INVITE', resolve => {
+    return (link: string) => resolve({ link })
+  }),
+  acceptExternalInviteSuccess: createAction('ACCEPT_EXTERNAL_THREAD_INVITE_SUCCESS', resolve => {
+    return () => resolve()
+  }),
+  acceptExternalInviteError: createAction('ACCEPT_EXTERNAL_THREAD_INVITE_ERROR', resolve => {
+    return (error: Error) => resolve({ error })
   })
```

Comment:
Feels good to say this for once... There are tests for `ThreadsRedux`, so please update the tests to cover this new stuff ;)

Reply:
will do